userModel.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const note = require('./noteModel');
+
+const UserSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true,
+    },
+    lastName: {
+        type: String,
+        required: true,
+    },
+    createdOn: {
+        type: Date,
+        required: true,
+        default: Date.now(),
+    }, 
+    notes: [
+        {
+            type: ObjectId,
+            ref: 'Note',
+        }
+    ]
+});
+
+const usersModel = mongoose.model('User', UserSchema);
```

Comment:
Word of caution: the disagreement in pluralization between `usersModel` and `UserSchema` will eventually generate a typo. I'd advise picking singular or plural and sticking with it, even if it causes odd grammar :)